osbuild/util/chroot.py

```diff
@@ -37,5 +37,13 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_value, tracebk):
+        failed_umounts = []
         for d in ["/proc", "/dev", "/sys"]:
-            subprocess.check_call(["/usr/bin/umount", self.root + d])
+            if subprocess.run(["/usr/bin/umount", self.root + d], check=False).returncode:
+                failed_umounts.append(d)
+        if failed_umounts:
+            print(f"Error unmounting paths from chroot: {failed_umounts}")
+
+    def run(self, cmd, **kwargs):
```

Comment:
Fwiw, not super important as it's indirectly tested already via dracut and also pretty simple but it does feel nice (I would offer to write if too next week in case you are too busy with other things :)

Reply:
I added a simple test that mocks the subprocess functions, counts invocations, and checks the first argument of each.